.yamato/config/hdrp.metafile

```diff
@@ -6,65 +6,64 @@ test_platforms:
   - type: playmode
   - type: playmode
     name: playmode_XR
-    extra_utr_flags:
-      - --extra-editor-arg="-xr-tests" 
+    utr_flags:
+      - [all]: --extra-editor-arg="-xr-tests" 
   - type: playmode
     name: playmode_NonRenderGraph
-    extra_utr_flags:
-      - --extra-editor-arg="-non-rendergraph-tests"
+    utr_flags:
+      - [all]: --extra-editor-arg="-non-rendergraph-tests"
   - type: editmode
   - type: Standalone
-    timeout:
-      OSX_Metal: 2400
+    utr_flags:
+      - [OSX_Metal]: --timeout=2400
 platforms:
   - name: Win
     apis:
-      - DX11
-      - DX12
-      - Vulkan
-    overrides:
-      agents_project:
-        # default: package_ci_mac
-        default:
-          type: Unity::VM::GPU
-          image: graphics-foundation/win10-dxr:stable
-          flavor: b1.xlarge
-          model: rtx2080
-        editmode:
-          type: Unity::VM
-          image: graphics-foundation/win10-dxr:stable
-          flavor: b1.xlarge
-        standalone:
-          type: Unity::VM::GPU
-          image: graphics-foundation/win10-dxr:stable
-          flavor: b1.xlarge
-          model: rtx2080
-        standalone_build:
-          type: Unity::VM
-          image: graphics-foundation/win10-dxr:stable
-          flavor: b1.xlarge
+      - name: DX11
+      - name: DX12
+      - name: Vulkan
+    build_configs:
+      - name: mono_apiNet2
+    color_spaces:
+      - Linear
```

Comment:
does this PR allow us to specify build configs and color spaces on a per-platform basis? so that we only generate the jobs we need?

Reply:
yep, now each platform has sections 'apis', 'build_configs', and 'color_spaces'. build_configs section only requires the name (actual config is in the shared file)

Reply:
amazing :D